changelog.md

```diff
@@ -2,6 +2,67 @@
 
 ## 0.13.0~rc1
 
+### Web Application
+
+* Added fingerprint of GPG reply key to Source object in Journalist API (#4436)
+* Updated message for flagged sources in Source Interface (#4428)
+* Added type hinting in Journalist Interface (#4404, #4407)
+* Added a /logout endpoint to the Journalist API (#4349)
+* Fixed 500 error caused by inconsistent session (#4391)
+* Added submission file sha256 hash as Etag on downloads in Journalist Interface (#4314)
+* Added Python3 compliance for Source and Journalist Interface (#4239)
+* Updated Python cryptography dependency, updated safety check (#4297)
+* Added support for HTTP DELETE method on Journalist Interface for use by API (#4023)
+* Removed cssmin dependency (#4227)
+* Removed jQuery dependency from Journalist and Source Interface (#4211)
+
+### Operations
+
+* Removed Trusty-specific code and configuration (#4422, #4416, #4348, #4311, #4224)
+* Consolidated CI lint Makefile targets (#4435)
+* Added 0.12.2  boxes for use with the Molecule upgrade scenario (#4393)
+* Added deb tests to builder image update (#4388)
+* Removed unused Jenkins configuration (#4337)
+* Added version pinning for Tor package fetch CI job (#4300)
+* Updated Tor version check to 0.3.5.8 (#4258)
+* Updated staging CI job timeout to 20 minutes (#4218)
+* Added CI job to run page layout tests in all supported languages for i18n-\* branches (#4184)
+
```

Comment:
It might be worth noting that we've completely eliminated Trusty support in this release (and that we've removed Trusty logic via #4341):

Reply:
Have a single catchall line now for the removal of Trusty including the relevant PRs in Operations section.